### DISCUSSION TITLE
organizzazione fisica - 3

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#82] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/ddfcc430-1dcc-421c-8a3a-5b0d7988033c)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-23 133535](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/d482954d-50b2-4ae1-aae5-8ff67191d74b)
![Immagine 2024-01-23 133548](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/fff1d7e6-b363-4c30-bf14-a1d5b0c8f066)
nell' ultima riga intendevo la radice caricata in Ram

**Reply by 7Andrei** [reply]
L0 non andrebbe calcolato con i RpB? e non con i record totali?

**Reply by Frisbon** [reply]
Si, penso ci sia un errore

**Reply by daromacarlo** [reply]
@7Andrei hai ragione

**Comment by Frisbon** [comment]
- NR = 1700000
- RS = 250
- KS = 45
- BS = 2048
- PS = 4
Minimo di accessi?
Record per blocco = <(BS/RS) = 8
Blocchi per file principale = >(NR/RpB) = 212500
Chiavi per nodo (m) = <[(BS-PS)/(KS+PS)] = <(2044/49) = 41
NumBlocchiLivello0 = >(212500/41) = 5183
(1) 5183/41 = (2) 127/41 = (3) 4/41 = (4) 1
Numero Accessi Minimo = 4+1 = 5

### poi vabbé ho questo scarabbocchio
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/3be27fac-6dc0-4fb2-9b71-4bacf502b9c4)
